aspnetcore/fundamentals/dotnet-scaffold-telemetry.md

```diff
@@ -0,0 +1,70 @@
+---
+title: dotnet-scaffold telemetry
+author: tdykstra
+description: Learn about the telemetry collected by the dotnet-scaffold CLI tool.
+monikerRange: '>= aspnetcore-8.0'
+ms.author: tdykstra
+ms.date: 11/06/2024
+uid: fundamentals/dotnet-scaffold-telemetry
+---
+# dotnet-scaffold telemetry
+
+The dotnet-scaffold command includes a telemetry feature that collects usage data. This feature helps the dotnet-scaffold team understand how the tool is used so it can be improved.
```

Comment:
I like the idea of "The `dotnet-scaffold` command includes ..." instead of what it looks like rn. I think the tool name should stand out, how do you feel about that?

Reply:
Yes, that's what we do for the built-in dotnet commands such as [dotnet build](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-build), so we should do that here. The httprepl docs, don't do that but I'll write up an issue to reformat those so they do it the same way as the other dotnet tool docs.